tutorial/simple-todos/02-collections.md

```diff
@@ -62,13 +64,15 @@ Now comes the fun part, you will render the tasks saved in our database. With Sv
 
 On your file `App.svelte`, import the `TasksCollection` file and, instead of returning a static array, return the tasks saved in the database. Let's use an extension of the Svelte's [$ reactive statements](https://svelte.dev/docs#3_$_marks_a_statement_as_reactive) feature, to maintain your tasks, called [$m](https://github.com/zodern/melte#tracker-statements):
 
+Because it is an asynchronous function, we need to use the `#await` keyword to wait for the data to be fetched from the database. More information about how Svelte handles asynchronous values can be found [here](https://svelte.dev/docs#template-syntax-await).
```

Comment:
It is easy to misunderstand this as meaning it waits for the data to be fetched by the server. Instead, `fetchAsync` will almost immediately resolve with whatever data the client currently has, and when the data changes a new promise will be created that resolves with the new data.

Reply:
So this paragraph is misleading? I thought that this could be a good way to show how to handle these cases

Reply:
It is a good explanation. It maybe just needs to clarify what is meant by wait and the database. When I first read it, I thought it was saying it was waiting for the complete set of data to be retrieved from the server. Part of the confusion might also be from that is what promises are usually used for, and what the promise is for here is different (`fetchAsync` isn't waiting for anything to happen. The only reason it returns a promise is to be consistent with the server).